Index: src/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>public class Main {\n        int year = 2023;\n    public static int sumOfTwoNumbers (int a, int b) {\n        int c = a + b;\n        return c;\n    }\n    public static void main(String[]args){\n        sumOfTwoNumbers(2,3);\n    }\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Main.java b/src/Main.java
--- a/src/Main.java	
+++ b/src/Main.java	
@@ -1,10 +1,12 @@
 public class Main {
-        int year = 2023;
-    public static int sumOfTwoNumbers (int a, int b) {
-        int c = a + b;
-        return c;
-    }
-    public static void main(String[]args){
-        sumOfTwoNumbers(2,3);
+
+    public static void main(String[] args) {
+
+        H3Month h3MonthObject = new H3Month();
+        int numberOfDays = h3MonthObject.getNumberOfDays(6);
+
+        System.out.println("Кількість днів у місяці: " + numberOfDays);
     }
 }
+
+
